caravel/assets/javascripts/explorev2/components/ChartControl.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Select from 'react-select';
+import { bindActionCreators } from 'redux';
+import * as actions from '../actions/exploreActions';
```

Comment:
rather than importing all actions here i would suggest just importing the ones needed for this component...
`import { setFormOpts, setDatasource, resetFormData, setVizType } as actions from '../actions/exploreActions';`